src/main/java/com/erigitic/commands/SetBalanceCommand.java

```diff
@@ -97,27 +99,24 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
     /**
      * Get the transaction result returned from setting a player's balance.
      *
-     * @param recipientAccount The account
-     * @param amount Amount of currency involved in the transaction
-     * @param optCurrencyName Currency name
+     * @param recipient    The account
```

Comment:
Don't worry about lining up the descriptions of the params. One space after the param name will suffice.

Reply:
Interesting. I don't even remember doing that ^^.  
Whatever - will change that anyways :D